frontend/src/lib/components/apps/editor/ComponentHeader.svelte

```diff
@@ -78,6 +85,7 @@
 					<Alert type="error" title="Error during execution">
 						<div class="flex flex-col">
 							<span> See "Debug Runs" on the top right for more details </span>
```

Comment:
I would remove this altogether and put the error directly, with the button underneath or above it's clear enough

Reply:
Done